html/themes/custom/common_design_subtheme/components/hri-toc/hri-toc.js

```diff
@@ -31,6 +31,15 @@
         }
       });
 
+      if (toc.length <= 1) {
+        // Remove toc.
+        targets.forEach(function (target) {
+          target.parentElement.remove();
+        });
+
+        return;
+      }
+
```

Comment:
too clever! putting me out of a job here

Reply:
Improved, so sidebar gets removed as well when it becomes empty ;-)

Reply:
ooh now we're getting spicy 🌶️